ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupDemoController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+// MARK: - AvatarGroupDemoController
+
+class AvatarGroupDemoController: DemoController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addTitle(text: "Avatar Stack")
+        insertAvatarViews(style: .stack, showBorders: false)
+        addTitle(text: "Avatar Pile")
```

Comment:
shouldn't it also add without border to match the old version?

Reply:
Yes I didn't add the border changes per our conversation offline earlier. I'll add that to the description as well!